src/containers/Profile.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <div class="profileContainer">
+    <div class="profileContainer-header">
+      <div class="profileContainer-title">
+        <span> Edit Profile</span>
+      </div>
+      <div class="profileContainer-subtitle">
+        <span> get your information up-to-date! </span>
```

Comment:
Is `Get` supposed to be lower case?

Reply:
the figma design had it lower case so i kept it as is